lib/test-worker.js

```diff
@@ -62,7 +62,7 @@ sourceMapSupport.install({
 
 var loudRejection = require('loud-rejection/api')(process); // eslint-disable-line
 var serializeError = require('./serialize-error');
-var send = require('./send');
```

Comment:
Could you elaborate? I wouldn't call a line, which consists of variable declaration and a require statement, a complication.